framework/packages/abstract-sdk/src/base/features/executor.rs

```diff
@@ -20,3 +20,5 @@ pub trait AccountExecutor: AccountIdentification {
             .map_err(Into::into)
     }
 }
+
+impl<T> AccountExecutor for T where T: AccountIdentification {}
```

Comment:
I think this is interesting to add, so added here. There is no reason to re-implement that explicitely on structures that have AccountIdentification

Reply:
Thanks for bringing attention to that change, unfortunately I think we would want to revert it. The point of implementing that explicitly is to limit which contract can and which cannot execute as account. Unless we have a better way to limit it with types?

Reply:
I'm not sure what you mean here. That `Standalone` can't execute as the account ?

Reply:
Yup, correct. We want to implement AccountExecutor on App and Adapter but not implement it for Standalone because in the install flow a standalone is not added to the permission set of account.